sentry/src/main/java/io/sentry/SentryOptions.java

```diff
@@ -1278,6 +1306,60 @@ public boolean isTracingEnabled() {
     return getTracesSampleRate() != null || getTracesSampler() != null;
   }
 
+  /**
+   * Returns the list of exception classes that once captured will not be sent to Sentry as {@link
+   * SentryEvent}.
+   *
+   * @return the list of exception classes that once captured will not be sent to Sentry as {@link
+   *     SentryEvent}.
+   */
+  public @Nullable Set<Class<? extends Throwable>> getIgnoredExceptionsForType() {
+    return ignoredExceptionsForType;
+  }
+
+  /**
+   * Sets the list of exception types that once captured will not be sent to Sentry as {@link
+   * SentryEvent}.
+   *
+   * @param ignoredExceptionsForType list of ignored exception types
+   */
+  @SuppressWarnings("unchecked")
+  public void setIgnoredExceptionsForType(final @NotNull Set<Class<?>> ignoredExceptionsForType) {
```

Comment:
we don't offer a set for any other array/list, but they should call `addIgnoredExceptionForType` instead so if at some point we come with some default types, they don't get overwritten

Reply:
This is an edge case how Spring Boot binds properties. If the property is a collection and is initialized in a field or constructor, it calls `collection#add()` - in this case it would mean that it's technically possible to set types that do not extend throwable into this list.

Reply:
I'll take another look if there is a better way to overcome it